nautobot/core/views/__init__.py

```diff
@@ -478,3 +478,9 @@ def get_file_with_authorization(request, *args, **kwargs):
     get_object_or_404(queryset, file=request.GET.get("name"))
 
     return get_file(request, *args, **kwargs)
+
+
+class RenderJinjaView(TemplateView):
```

Comment:
Does this give appropriate permission constraints (at least `is_authenticated`)?

Reply:
Nope I totally forgot about auth for this view! Will fix it. I think `is_authenticated` will be ok for this one since this is just calling the REST API and we can do more granular permissions there if necessary?